testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/containers/KeycloakQuarkusServerDeployableContainer.java

```diff
@@ -172,7 +172,7 @@ private Process startContainer() throws IOException {
             commands.add("-Djboss.node.name=" + configuration.getRoute());
         }
 
-        commands.add("--cluster=" + System.getProperty("auth.server.quarkus.cluster.config", "local"));
+        commands.add("--cache-config-file=cache-" + System.getProperty("auth.server.quarkus.cluster.config", "local") + ".xml");
```

Comment:
just a nitpick: could you also rename the system property to be `auth.server.quarkus.cache.config` to stay consistent?

Reply:
Yeah, should be fixed now.